db/app.js

```diff
@@ -1,7 +1,46 @@
 const express = require("express");
 const app = express();
 app.use(express.json());
-const getApi = require("../db/seeds/api.controller");
+const {
+  getApi,
+  getTopics,
+  getArticleById,
+  getArticle,
+} = require("../db/seeds/api.controller");
 console.log("I am in app");
+
 app.get("/api", getApi);
+
+app.get("/api/topics", getTopics);
+
+app.get("/api/articles", getArticle);
```

Comment:
If this is for Task 4, getArticles, a more semantic name for this would be to pluralise the function.